src/relay/transforms/simplify_expr.cc

```diff
@@ -878,6 +932,33 @@ class SimplifyAdd : public DFPatternRewrite {
   DFPattern y_;
 };
 
+/*! \brief Simplifying x/c to x*(1/c) */
+class SimplifyDivide : public DFPatternRewrite {
+ public:
+  SimplifyDivide() {
+    x_ = IsWildcard();
+    c_ = IsConstant();
+    pattern_ = IsOp("divide")({x_, c_});
+  }
+
+  Expr Callback(const Expr& pre, const Expr& post,
+                const Map<DFPattern, Array<Expr>>& node_map) const override {
+    Type pre_type = pre->checked_type_;
+    auto dtype = pre_type.as<TensorTypeNode>()->dtype;
+    auto x = node_map[x_][0];
+    auto c = node_map[c_][0];
+
+    Expr one;
+    one = MakeConstantScalar(dtype, 1);
```

Comment:
Minor: var declare and initialize at the same time.

Reply:
fixed